AUTHORS

```diff
@@ -757,3 +757,35 @@ Ayushman Koul <ayushmankoul4570@gmail.com>
 Robert Dougherty-Bliss <robert.w.bliss@gmail.com>
 Andrey Grozin <A.G.Grozin@inp.nsk.su>
 Bavish Kulur <bavishkulur@gmail.com>
+Arun Singh <arunsin997@gmail.com>
+sirnicolaf <43586954+sirnicolaf@users.noreply.github.com>
+Zachariah Etienne <zachetie@gmail.com>
+Prayush Dawda <35144226+iamprayush@users.noreply.github.com>
+2torus <boris.ettinger@gmail.com>
+Faisal Riyaz <faisalriyaz011@gmail.com>
+Martin Roelfs <u0114255@kuleuven.be>
+SirJohnFranklin <sirjfu@googlemail.com>
+Anthony Sottile <asottile@umich.edu>
+ViacheslavP <public.viacheslav@gmail.com>
+Safiya03 <safiyanesar@gmail.com>
+Alexander Dunlap <alexander.dunlap@gmail.com>
+Rohit Sharma <31184621+rohitx007@users.noreply.github.com>
+Jonathan Warner <warnerjon12@gmail.com>
+Mohitbalwani26 <44258119+Mohitbalwani26@users.noreply.github.com>
+Marduk Bolaños <mardukbp@mac.com>
+amsuhane <ayushsuhane99@iitkgp.ac.in>
+Matthias Geier <Matthias.Geier@gmail.com>
+klaasvanaarsen <44929042+klaasvanaarsen@users.noreply.github.com>
+Shubham Kumar Jha <skjha832@gmail.com>
+rationa-kunal <kunalgk1999@gmail.com>
+Animesh Sinha <animeshsinha1309@gmail.com>
+Gaurang Tandon <1gaurangtandon@gmail.com>
+Matthew Craven <clyring@users.noreply.github.com>
+Daniel Ingram <ingramds@appstate.edu>
+Jogi Miglani <jmig5776@gmail.com>
+Takumasa Nakamura <n.takumasa@gmail.com>
+RituRajSingh878 <RituRajSingh878@gmail.com>
+Rajiv Ranjan Singh <42106787+iamrajiv@users.noreply.github.com>
```

Comment:
These two are duplicates. Are you using the authors script to generate this file?

Reply:
yes,  I am using authors script to generate this file.

Reply:
> These two are duplicates.

which one?

Reply:
Oh sorry, I thought Rajiv Ranjan Singh and you were the same person (you have very similar names).